tool/mesos/singa.conf

```diff
@@ -0,0 +1,7 @@
+# point to your active zookeeper service
```

Comment:
this file is duplicated from `incubator-singa/conf/singa.conf`.
we can use that file and add some comments for configuring the zookeeper host using docker node.